### PR TITLE
#9 created login and signup pages

### PR DESCRIPTION
**Pull request by quynhngandao**

### CODE REVIEW

File: app/login.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Link } from 'expo-router';
```
Comment (michael-duren) [inline app/login.tsx]: Can remove unecessary imports.

File: components/features/Auth/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            <TextInput
+              className="w-full py-3 pl-12 bg-gray-200 rounded-lg focus:outline-none"
+              onChangeText={(text) => setUsername(text)}
+              value={username}
+              placeholder="Username"
+              id="username"
+            />
+          </View>
+          <View className="flex items-center mb-4 text-lg ">
+            <Svg
```
Comment (michael-duren) [inline components/features/Auth/LoginForm.tsx]: Can we seperate the SVGs into a `Icons.ts` file? Or use the icons from the react-native-vector-icons?

File: package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
+    "react-native-svg": "^14.0.0",
```
Comment (michael-duren) [inline package.json]: I'm getting a good amount of errors from this, I'm not sure if it's this package or not though.

**Review by michael-duren — CHANGES_REQUESTED** [review]
There seems to be some issues when I try to run this on my machine. I will post screen shots in slack.

File: components/features/Auth/LoginForm.tsx (hunk 2 above)
Comment (quynhngandao) [reply components/features/Auth/LoginForm.tsx]: Replaced with react-native-vector-icons

**Comment by quynhngandao** [comment]
Replaced the svg to vector-icons

**Review by michael-duren — APPROVED** [review]
Nice job!

**Pull request merged**